apps/stress-test/package.json

```diff
@@ -0,0 +1,68 @@
+    "bench:simple-stress:mount:chrome": "tach --config ./benchmarks/simple-stress/config/chrome.mount.json --json-file ./benchmarks/simple-stress/results/chrome.mount.results.json",
+    "bench:simple-stress:mount:firefox": "tach --config ./benchmarks/simple-stress/config/firefox.mount.json --json-file ./benchmarks/simple-stress/results/firefox.mount.results.json",
+    "bench:simple-stress:inject-styles:chrome": "tach --config ./benchmarks/simple-stress/config/chrome.inject-styles.json --json-file ./benchmarks/simple-stress/results/chrome.inject-styles.results.json",
+    "bench:simple-stress:inject-styles:firefox": "tach --config ./benchmarks/simple-stress/config/firefox.inject-styles.json --json-file ./benchmarks/simple-stress/results/firefox.inject-styles.results.json",
+    "bench:simple-stress:prop-update:chrome": "tach --config ./benchmarks/simple-stress/config/chrome.prop-update.json --json-file ./benchmarks/simple-stress/results/chrome.prop-update.results.json",
+    "bench:simple-stress:prop-update:firefox": "tach --config ./benchmarks/simple-stress/config/firefox.prop-update.json --json-file ./benchmarks/simple-stress/results/firefox.prop-update.results.json",
+    "bench:simple-stress:add-node:chrome": "tach --config ./benchmarks/simple-stress/config/chrome.add-node.json --json-file ./benchmarks/simple-stress/results/chrome.add-node.results.json",
+    "bench:simple-stress:add-node:firefox": "tach --config ./benchmarks/simple-stress/config/firefox.add-node.json --json-file ./benchmarks/simple-stress/results/firefox.add-node.results.json",
+    "bench:simple-stress:remove-node:chrome": "tach --config ./benchmarks/simple-stress/config/chrome.remove-node.json --json-file ./benchmarks/simple-stress/results/chrome.remove-node.results.json",
+    "bench:simple-stress:remove-node:firefox": "tach --config ./benchmarks/simple-stress/config/firefox.remove-node.json --json-file ./benchmarks/simple-stress/results/firefox.remove-node.results.json"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.10.4",
+    "@babel/plugin-proposal-decorators": "^7.18.10",
+    "@babel/preset-env": "7.14.8",
+    "@babel/preset-react": "7.14.5",
+    "@babel/preset-typescript": "^7.18.6",
+    "@griffel/webpack-extraction-plugin": "^0.1.4",
+    "@griffel/webpack-loader": "2.0.12",
+    "@types/node": "10.17.55",
+    "@types/react": "16.14.23",
+    "@types/react-dom": "16.9.14",
+    "babel-loader": "8.2.2",
+    "chromedriver": "^104.0.0",
+    "clean-webpack-plugin": "3.0.0",
+    "commander": "^9.4.0",
+    "cross-env": "^5.1.4",
+    "css-loader": "5.0.1",
+    "geckodriver": "^3.0.2",
+    "html-webpack-plugin": "5.1.0",
+    "http-server": "^14.1.1",
+    "mini-css-extract-plugin": "^2.6.1",
+    "tachometer": "^0.7.0",
+    "terser-webpack-plugin": "^5.3.3",
+    "ts-loader": "8.0.14",
+    "typescript": "4.3.5",
+    "webpack": "^5.21.2",
+    "webpack-cli": "4.7.2",
+    "webpack-dev-server": "4.7.4"
+  },
```

Comment:
@Hotell AFAIK all `devDependencies` should go to a root `package.json`, is it correct?

Reply:
Done!